files/tachyons/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "tachyons",
+  "repo": "mrmrs/tachyons",
+  "files": {
+    "include": ["css/tachyons.css","css/tachyons.min.css"]
```

Comment:
I dont think the unminified file is needed in production. You can just leave the min